tests/Repositories/MadRepositoryTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace EscolaSoft\KibbleMad\Tests\Repositories;
```

Comment:
We definitly have to remove that namespace and class ;)

Reply:
This file is removed